### PR TITLE
pdfcpu: init at 0.1.25

### PR DESCRIPTION
**Pull request by doronbehar**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"` - non depend upon it.
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/graphics/pdfcpu/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0vmmc7nnvpvsf92yi69rvqif1irkpya2shqyz49sa3s42jh1446b";
+  };
+
+  modSha256 = "0cz4gs88s9z2yv1gc9ap92vv2j93ab6kr25zjgl2r7z6clbl5fzp";
+
+  subPackages = [ "cmd/pdfcpu" ];
+
+  outputs = [ "out" ];
```
Comment (kalbasit) [inline pkgs/applications/graphics/pdfcpu/default.nix]:
```suggestion
```

**Review by kalbasit — CHANGES_REQUESTED** [review]
PR title, and commit message for the package should be `pdfcpu: init at 0.1.25`

File: pkgs/applications/graphics/pdfcpu/default.nix (hunk 1 above)
Comment (doronbehar) [reply pkgs/applications/graphics/pdfcpu/default.nix]: Right, fixed in forced push.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -1353,6 +1353,11 @@
     github = "domenkozar";
     name = "Domen Kozar";
   };
+  doronbehar = {
+    email = "me@doronbehar.com";
+    github = "doronbehar";
+    name = "Doron Behar";
+  };
```
Comment (kalbasit) [inline maintainers/maintainer-list.nix]: You had it correct earlier, this should remain in a separate commit. Also PR title needs updating, thx!

**Review by kalbasit — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (doronbehar) [reply maintainers/maintainer-list.nix]: The two separate commits I've had earlier were: 1. maintainer-list and new pdfcpu/default.nix and 2. Update license.

I've uploaded a forced push of first the maintainer update and then the `pdfcpu/default.nix` addition.

**Comment by kalbasit** [comment]
@GrahamcOfBorg build pdfcpu

**Review by kalbasit — APPROVED** [review]

**Comment by doronbehar** [comment]
Thanks!

**Pull request merged**